quicksave.lua

```diff
@@ -42,16 +34,25 @@ local function restore_autobackup()
 end
 
 function save()
-    -- Request auto-save
+    -- Request auto-save (preparation steps below discovered from rev eng)
     ui_main.autosave_request = true
+    ui_main.autosave_unk = 5
```

Comment:
I'm probably going to push a structures change to rename this field since I have a good idea what it does (although not why it does what it does)

and of course this violates our "rule" about not using `unk` or `anon` fields in scripts

Reply:
according to Putnam, `autosave_unk` is `auto_save_timer`

and `unk_v50_6` is `info`, if you'd like to make that change too while you're in there